packages/manager/src/features/CloudPulse/shared/CloudPulseResourcesSelect.tsx

```diff
@@ -118,6 +136,29 @@ export const CloudPulseResourcesSelect = React.memo(
         placeholder={
           selectedResources?.length ? '' : placeholder || 'Select Resources'
         }
+        renderOption={(props, option) => {
+          const { key, ...rest } = props;
+          const isResourceSelected = selectedResources?.some(
+            (item) => item.label === option.label
+          );
+          const isOverLimitOption =
+            selectedResources &&
+            selectedResources?.length >= setResourceLimit &&
+            !isResourceSelected;
+          return (
+            <ListItem
+              {...rest}
+              aria-disabled={isOverLimitOption}
+              data-qa-option
+              key={key}
+            >
+              <>
```

Comment:
Do we require this fragment <> here?

Reply:
we just picked up the renderOp prop from our reusable component and added one extra element aria-disabled, rest is same as autocomplete, i.e. we are just grouping in one element under listitem..

Reply:
here Listitem itself acts like a wrapper right, so.. Anyway if its already used and approved, then fine.